### PR TITLE
Anika relevant

### PR DESCRIPTION
**Pull request by anikarabenhorst**
Adds RelevantRightNow component and uses data from AirTable
## Notes: 
* Horizontal scroll for when there are more than 3
* date formatting may change based on some work with time zones being done by Dania
* if you use the code now, all relevant data will be filtered out because the Date has been changed to current time
* I used a date-time during the event to grab screenshots
Screen Shot 2021-01-28 at 3 41 22 PM
Screen Shot 2021-01-28 at 4 30 25 PM

### CODE REVIEW

File: src/styles/relevant.scss [hunk 1]
```diff
@@ -0,0 +1,40 @@
+.relevant {
+  .card-title {
+    font-size: 24px;
+    font-weight: bold;
+    margin-bottom: 4px;
```
Comment (snwessel) [inline src/styles/relevant.scss]: Would you mind adding a tiny bit more spacing below the title and above the text (to give the date and time a little more space)?

File: src/styles/main.scss [hunk 2]
```diff
@@ -8,13 +8,11 @@
 // Components
 @import "./navbar.scss";
 @import "./countdown.scss";
-@import "./resources.scss";
-@import "./mentors.scss";
```
Comment (snwessel) [inline src/styles/main.scss:12]: Oh good catch!

**Review by snwessel — APPROVED** [review]
This looks great to me! I just left one comment on spacing. 

Thanks for all your hard work on this!

File: src/components/RelevantRightNow.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  return start == null
+    ? `Complete by ${endString}`
+    : `${startString} - ${endString}`;
+}
+
+function isTimeBetween(start, end) {
+  // used different date to capture screenshots
+  // using now means all will be filtered out
+  const currentTime = new Date();
+  return isWithinInterval(currentTime, {
```
Comment (nupur-n) [inline src/components/RelevantRightNow.js:53]: this is great! will also be useful for mentor shifts <3

**Review by nupur-n — APPROVED** [review]
Awesome! Noting some styling things for next steps:
- set a min-height for cards so they're all the same size
- some way to indicate people need to scroll?

File: src/styles/main.scss (hunk 2 above)
Comment (anikarabenhorst) [reply src/styles/main.scss:12]: Screen Shot 2021-01-28 at 11 21 53 PM

File: src/styles/relevant.scss (hunk 1 above)
Comment (anikarabenhorst) [reply src/styles/relevant.scss]: Screen Shot 2021-01-28 at 11 22 30 PM

**Comment by anikarabenhorst** [comment]
> * some way to indicate people need to scroll?

@nupur-n i will ask Raquel if she has any thoughts

**Comment by nupur-n** [comment]
@anikarabenhorst awesome! def can be a separate PR, this is great ~~

**Pull request merged**